pkg/vips/bridge.go

```diff
@@ -262,9 +265,14 @@ func vipsDetermineImageType(buf []byte) ImageType {
 	if IsTypeSupported(ImageTypeWEBP) && buf[8] == 0x57 && buf[9] == 0x45 && buf[10] == 0x42 && buf[11] == 0x50 {
 		return ImageTypeWEBP
 	}
-	if IsTypeSupported(ImageTypeSVG) && buf[0] == 0x3c && buf[1] == 0x3f && buf[2] == 0x78 && buf[3] == 0x6d {
+	if IsTypeSupported(ImageTypeSVG) && buf[0] == 0x3C && buf[1] == 0x3F && buf[2] == 0x78 && buf[3] == 0x6D {
```

Comment:
We can replace the four single-byte comparisons with one int comparison to 0x6D783F3C. To convert the first four bytes of the buffer to int see https://stackoverflow.com/questions/11184336/how-to-convert-from-byte-to-int-in-go-programming

Reply:
You'll see that in my addition I use runes

Reply:
That's nicer